### DISCUSSION TITLE
Allow UI changes on provisioned alerts

### DISCUSSION THREAD

**Discussion by arash-bizcover** [grafana/grafana#57314] [Alerting]
Currently, the objects (alerting rules) created via the provisioning API are not editable from UI.
This behaviour should also be changed so that we can edit (provisioned) rules from UI and then export the json back to the provisioning code/file.

**Comment by arash-bizcover** [comment]
https://github.com/grafana/grafana/issues/57315

**Comment by JohnnyQQQQ** [comment]
Duplicate of https://github.com/grafana/grafana/discussions/57136
